ground/src/androidTest/java/com/google/android/ground/TestAuthenticationModule.java

```diff
@@ -18,7 +18,7 @@
 
 import com.google.android.ground.system.auth.AuthenticationManager;
 import com.google.android.ground.system.auth.AuthenticationModule;
-import com.google.android.ground.system.auth.FakeAuthenticationManager;
+import com.google.android.ground.test.system.auth.FakeAuthenticationManager;
```

Comment:
Do we need to use a different package name? One advantage to having test classes be in the same package as the unit under test is that they can see package-private variables and methods, which in some cases makes things easier to mock (e.g., overriding methods in test fakes).

Reply:
See https://github.com/google/ground-android/pull/1187#issuecomment-1207335080

With same package name, all instrumentation tests were failing due to unresolved resource ID.